src/dataDisplay/Icon/images/bookmark.tsx

```diff
@@ -12,7 +12,7 @@ export default {
         <path
           className="icon-stroke"
```

Comment:
Could you kill the classnames since you're at it? :)

Reply:
do you mean adding the fill directly in the svgs?

Reply:
Oh, I thought this class doesn't do anything. If it sets the color globally, please ignore my comment.